src/shim/kernel.rs

```diff
@@ -1,5 +1,18 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
+//! We have three goals for our error shims:
+//! - preserve upstream error _numbers_.
+//! - preserve upstream error messages.
+//! - allow _matching_ on specific errors.
+//!
+//! There are a couple of things that make this difficult:
+//! - `fvm_shared*::error::ErrorNumber` is `#[non_exhaustive]`
+//! - ...
```

Comment:
Remove this or fill it in - it was an example message :)

Reply:
I meant remove the `...` on line 10, not the entire docs, which I think you should keep :)

Reply:
Fixed.